### PR TITLE
Remove redundant functions and TxCallable.

### PR DESCRIPTION
**Pull request by chtyim**

### CODE REVIEW

**Comment by chtyim** [comment]
Build passed https://builds.cask.co/browse/CDAP-DUT6167-3

File: cdap-common/src/main/java/co/cask/cdap/common/security/YarnTokenUtils.java [hunk 1]
```diff
@@ -96,7 +96,7 @@ public static Credentials obtainToken(YarnConfiguration configuration, Credentia
           services.add(SecurityUtil.buildTokenService(YarnUtils.getRMAddress(configuration)).toString());
         }
 
-        Token<TokenIdentifier> token = ConverterUtils.convertFromYarn(rmDelegationToken, null);
+        Token<TokenIdentifier> token = ConverterUtils.convertFromYarn(rmDelegationToken, (InetSocketAddress) null);
```
Comment (sagarkapare) [inline cdap-common/src/main/java/co/cask/cdap/common/security/YarnTokenUtils.java:99]: any reason this type casting is required?

Comment (chtyim) [reply cdap-common/src/main/java/co/cask/cdap/common/security/YarnTokenUtils.java:99]: In later hadoop version, the the `convertFromYarn` mehtod has overload. Pass in `null` causes compilation error.

Comment (sagarkapare) [reply cdap-common/src/main/java/co/cask/cdap/common/security/YarnTokenUtils.java:99]: I see ok.

**Review by sagarkapare — APPROVED** [review]
LGTM 👍

**Comment by chtyim** [comment]
Squashed and merging.

**Pull request merged**